src/views/connectionUI.ts

```diff
@@ -441,6 +466,18 @@ export class ConnectionUI {
         });
     }
 
+    private promptForRetryConnectWithDifferentCredentials(): PromiseLike<boolean> {
+        // Ask if the user would like to fix the profile
+        return this._vscodeWrapper.showErrorMessage(LocalizedConstants.msgPromptRetryConnectionDifferentCredentials
+            , LocalizedConstants.retryLabel).then(result => {
```

Comment:
Minor: I'd just put this on 1 line even if it's a bit long

Reply:
the long is too long so I get formatting error if I don't put them in to lines

---
In reply to: [140110249](https://github.com/Microsoft/vscode-mssql/pull/993#discussion_r140110249) [](ancestors = 140110249)